# HG changeset patch
# User jwe
# Date 748051743 0
#      Wed Sep 15 00:09:03 1993 +0000
# Node ID ae8f1ad2774ae3fc49e8b39226f2a61f53fcfe27
# Parent  38f924cd2055ccb615850dbb68571c0bb09109a0
[project @ 1993-09-15 00:09:03 by jwe]
Use err, not info for fsolve section.

diff --git a/scripts/general/perror.m b/scripts/general/perror.m
--- a/scripts/general/perror.m
+++ b/scripts/general/perror.m
@@ -18,23 +18,23 @@ function perror (name, err)
   endif
 
   if (! is_scalar (err))
     error ("perror: second argument must be a scalar");
   endif
 
   if (strcmp (name, "fsolve"))
 
-    if (info == -1)
+    if (err == -1)
       printf ("input error\n");
-    elseif (info == 1)
+    elseif (err == 1)
       printf ("solution converged to requested tolerance\n");
-    elseif (info == 4)
+    elseif (err == 4)
       printf ("iteration limit exceeded\n");
-    elseif (info == 3)
+    elseif (err == 3)
       printf ("iteration is not making good progress\n");
     else
       error ("perror: unrecognized error code for fsolve");
     endif
 
   elseif (strcmp (name, "npsol"))
 
     if (err == 0)
